svcomp.properties

```diff
@@ -0,0 +1,4 @@
+method=eager
+encoding.integers=true
```

Comment:
I thought you intend to use BVs and Yices2, no? Or is this different to the settings you will actually submit?